playbooks/adhoc/upgrades/upgrade.yml

```diff
@@ -311,7 +311,7 @@
     openshift_master_ha: "{{ groups['masters'] | length > 1 }}"
   tasks:
     - name: Restart master services
-      service: name="{{ openshift.common.service_type}}-master" state=restarted
+      service: name="{{ openshift.common.service_type}}-master" enabled=yes state=restarted
```

Comment:
I'd prefer not to set enabled=yes here, since we are mainly restarting to pick up the new settings...  perhaps we should move this to a handler and have the steps that need a restart notify the handler?